tools/test

```diff
@@ -162,6 +171,22 @@ run_prettier() {
       "${patterns[@]}"
 }
 
+run_deps() {
+    files_check package.json yarn.lock \
+        || return 0
+
+    if ! yarn-deduplicate --fail --list; then
+        cat >&2 <<EOF
+
+Duplicates could be shared in yarn.lock.
```

Comment:
This error message is more than a little cryptic; I'd suggest something more explicit, along the lines of "`Mergeable duplicate dependencies found in yarn.lock`".

Reply:
Good idea. I went for even more explicit than that:
> Found duplicate dependencies in yarn.lock which could be dedup'd.